dhall-yaml/README.md

```diff
@@ -13,7 +13,16 @@ Full documentation here:
 This `dhall-yaml` package provides a Dhall to YAML compiler, `dhall-to-yaml-ng`,
 and a tool for deriving Dhall from YAML code: `yaml-to-dhall`.
 
-Tutorials for the analogous JSON tools are available in the `dhall-json` package:
+Note that the `dhall-json` package also provides a `dhall-to-yaml` executable.
+The main benefit of this package is that it depends on `HsYAML` (a pure Haskell
+implementation of YAML) instead of the `yaml` package (which depends on the
+C `libyaml` package).  This means, for example, that this package can be built
```

Comment:
No, the `yaml` dependency is actually long gone. `dhall-to-yaml` uses `aeson-yaml` now, which is pure Haskell too.

AFAIK the current main difference is the licence, and there are some minor differences in the output formatting. I think the testsuite documents that to some extent:

https://github.com/dhall-lang/dhall-haskell/blob/8effcf2474363466718bd7256cfc10802cc0ea0c/dhall-yaml/tasty/Main.hs#L36-L66

More significant differences are planned, but not implemented yet. If we translate Dhall to YAML directly (probably using HsYAML), we could implement https://github.com/dhall-lang/dhall-haskell/issues/1379.